cdc/owner.go

```diff
@@ -205,11 +204,7 @@ func (o *Owner) newChangeFeed(
 		failpoint.Return(nil, errors.New("failpoint injected retriable error"))
 	})
 
-	// TODO here we create another pb client,we should reuse them
-	kvStore, err := kv.CreateTiStore(strings.Join(o.pdEndpoints, ","), o.credential)
-	if err != nil {
-		return nil, err
-	}
+	kvStore := util.KVStorageFromCtx(ctx)
```

Comment:
Is kvstorage thread-safe?

Reply:
yes, and the recommended usage is to use him as a singleto.

Reply:
So what about a global variable?